modular_skyrat/modules/customization/modules/language/russian.dm

```diff
@@ -1,5 +1,5 @@
 /datum/language/russian
-	name = "Russian"
+	name = "Neo-Russkya"
```

Comment:
```suggestion
	name = "Neo-Russian"
```
Every other language save for like, the demon one, is in English.

Reply:
This reminds me, maybe make it Pan-Slavic instead. It's more likely for a good portion of eastern european languages to merge with eachother and become bastardized over the years than for Russian to be "adapted". Take galactic common as an example, it's a bastard mix of chinese/french/spanish/english etc.